File: Java/Java/PerfectNumber.java
import java.util.Scanner;

public class PerfectNumber {
    
    public static void main(String[] args) {
        
        int n,i,sum;
        Scanner s = new Scanner(System.in);
        
        System.out.print("Enter the Number : ");
        n = s.nextInt();
        
        i = 1;
        sum = 0;
        
        while(i<n)
        {
            if(n%i==0)
            {
                sum += i;
            } 
              i++;            
        }  
        
        if(sum==n) 
        {
            System.out.print(n+" this Number is Perfect Number");
        }  
        else
        {
            System.out.print(n+" this Number is Not Perfect Number");
        }            
        
        
    }
    
}

File: C/Perfect Number.c
#include<stdio.h>

int main()
{
    int n,i,sum=0;
    
    printf("Enter the Number: ");
    scanf("%d",&n);
    
    for(i=1;i<n;i++)
    {
      if(n%i==0)
     {
       sum=sum+i;
     }
    }
    
    if(sum==n)
    {
      printf("%d This is Perfect Number",n);
    }
    else
    {
       printf("%d This is Not Perfect Number",n);
    }
    return 0;
}